### ISSUE TITLE
Update to latest Kanbard version?

### ISSUE THREAD

**Issue by poVoq** [andrewshadura/kanboard-plugin-matrix#1]
Seems like this plugin isn't available through the plugin installer any more. Would be nice if it was made to work again.

**Comment by MayeulC** [comment]
I agree. Looks like it was removed from the plugin list for no really good reason: https://github.com/kanboard/kanboard/issues/3992

Although I wasn't able to make it work with a manual installation (and there's no mention of the plugin in the debug log).
